mix.exs

```diff
@@ -75,8 +75,10 @@ defmodule SHT4X.MixProject do
         "test",
         "mix.exs",
         "README.md",
-        "LICENSE",
-        "CHANGELOG.md"
+        "CHANGELOG.md",
+        "LICENSES",
+        "NOTICE",
+        "REUSE.toml"
       ],
       licenses: ["MIT"],
       links: %{
```

Comment:
Could you add "REUSE compliance" to the links? I have examples in other projects. Sadly, scripts don't add it. Please double check the URL. I think that I've messed up the URL many times due to copy/paste in projects and need to review projects to make sure it's correct.